lib/wam.coffee

```diff
@@ -197,27 +198,25 @@ getAddonMetadata = (addonIdentifier) ->
     downloadPageURL =
       "https://www.curseforge.com/wow/addons/#{addonIdentifier}/download"
 
-    request {
-      followAllRedirects: true,
-      url: downloadPageURL
-    }, (error, response) ->
-      if error
-        return reject "Error getting Addon information for
-                       #{addonIdentifier.yellow}"
+    cs.get downloadPageURL
+      .then (data) ->
+        linkRegex = new RegExp(
+          "/wow/addons/#{addonIdentifier}/download/[^\"]*/file"
```

Comment:
Any reason to remove the `href="` prefix?

Reply:
for more general cases, like
`Elerium.PublicProjectDownload.countdown("/wow/addons/deadly-boss-mods/download/2810713/file");`
and make regexp simpler and easy to maintain.